playground/package.json

```diff
@@ -2,16 +2,17 @@
     "private": true,
     "name": "lwc-playground",
     "version": "2.29.0",
+    "type": "module",
```

Comment:
This is the only package whose `rollup.config.js` was doing `import`s instead of `require`s. Hence this change is needed.

No other scripts in this package are affected.